### DISCUSSION TITLE
DatePicker 组件可以设置多个 format，请问是干嘛用的

### DISCUSSION THREAD

**Discussion by liuweiGL** [ant-design/ant-design#40647] [Q&A] [answered]
`format` 设置成多个没发现有什么不一样的地方啊？
```tsx
const dateFormatList = ['DD/MM/YYYY', 'DD/MM/YY'];
<DatePicker defaultValue={dayjs('01/01/2015', dateFormatList[0])} format={dateFormatList} />
```

**Comment by Wxh16144** [comment, marked as answer]
你可以将这个功能理解为提升用户输入体验的。在你提供的例子上我又给 `dateFormatList` 添加了两项。

```tsx
const dateFormatList = ["DD/MM/YYYY", "DD/MM/YY", "DD-MM-YYYY", "DD-MM-YY"];
```

`format` 为数组时，日期选择器默认显示 `format[0]` 格式。默认情况下日期选择器输入框是允许用户输入的。
这个时候用户就可以输入 `dateFormatList` 其中任意一种格式都可以正确的解析。如下视频：

(注意输入的格式和日期面板)


https://user-images.githubusercontent.com/32004925/217715545-9099709c-2ac0-4d8c-b005-cdb398e68f47.mp4

**Comment by liuweiGL** [comment]
棒棒的，Thanks♪(･ω･)ﾉ
